Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
> a middle-man between the FormMapper and the "real" Symfony FormBuilder

Hence the word "contractor", maybe ?

> I think this method belongs in fact to the FormContractorInterface, because it can be private to the other "normal" builders as no one calls it from the outside.

Should we make it private then? If yes, it should be removed from the interface, shouldn't it ?